### ISSUE TITLE
Writing to text file from execution module lacks eol char \n

### ISSUE THREAD

**Issue by gerardjp** [saltstack/salt#18552]
Hi All,

I see different behaviour between a simple commandline script and a Salt execution module while execution this code snippet:

```
    with open(flag_file, 'a') as f:
        f.write(attrib+'\n')
        f.close()
```

flag_file's contents via module:

```
FLAG_AFLAG_B
```

What I expected, and see when performed in a simple commandline script, is this:
```
FLAG_A
FLAG_B
```

Am I missing something obvious or is this deviate behaviour?

Thanx a lot.

Kind regards,

Gerard.

**Comment by rallytime** [comment]
Sorry we never got back to you here @gerardjp. I am not sure if this has been resolved or not, but would you mind trying this again on 2014.7.5 and see if you can reproduce? There were many bug fixes on 2014.7 since rc7, so this may already be fixed.

**Comment by rallytime** [comment]
I am going to close this since we haven't heard back at all. If this remains an issue, please leave a comment and we'll be happy to re-open.
